File: apps/s-2-admin/src/api/selectedCalendar/SelectedCalendarWhereInput.ts
import { IntFilter } from "../../util/IntFilter";
import { UserWhereUniqueInput } from "../user/UserWhereUniqueInput";
import { StringFilter } from "../../util/StringFilter";

export type SelectedCalendarWhereInput = {
  id?: IntFilter;
  user?: UserWhereUniqueInput;
  integration?: StringFilter;
  externalId?: StringFilter;
};

File: apps/cactus-demo-dotnet/src/APIs/SelectedCalendar/Dtos/SelectedCalendarWhereInput.cs
namespace CactusDemoDotnet.APIs.Dtos;

public class SelectedCalendarWhereInput
{
    public int? Id { get; set; }

    public int? User { get; set; }

    public string? Integration { get; set; }

    public string? ExternalId { get; set; }
}
